packages/SidePanel/src/components/Group/Group.styles.js

```diff
@@ -7,6 +7,7 @@ export const groupCSS = css`
   justify-content: flex-end;
   position: fixed;
   right: 0;
+  visibility: hidden;
```

Comment:
This is so that the <group> div is not visible, but the side panels inside it are. Then the overlay can be clickable even if the group has a `width: 100%` to support percentage widths for side panels.